installer.sh

```diff
@@ -85,20 +85,12 @@ else
   export TRAVIS_WORKER_VERSION
 fi
 
-if [[ ! -n $TRAVIS_BETA_BUILD_IMAGES ]]; then
-  export BUILD_IMAGES='trusty'
-else
+if [[ ! -n $TRAVIS_BUILD_IMAGES ]]; then
   export BUILD_IMAGES='xenial'
-
-  # Xenial workers listen to the builds.linux by default
```

Comment:
@schultyy I don't understand this line, unfortunately, so I don't know what the knock-on effects of removing this are. Maybe we can chat in the morning about it?